src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.Unix.cs

```diff
@@ -20,12 +20,17 @@ private static bool GetGlobalizationInvariantMode()
                 {
                     LoadAppLocalIcu(icuSuffixAndVersion);
                 }
-                else if (Interop.Globalization.LoadICU() == 0)
+                else if (Interop.Globalization.LoadICU() == 0 && !OperatingSystem.IsBrowser())
                 {
                     string message = "Couldn't find a valid ICU package installed on the system. " +
                                     "Set the configuration flag System.Globalization.Invariant to true if you want to run with no globalization support.";
                     Environment.FailFast(message);
                 }
+                else if (OperatingSystem.IsBrowser())
```

Comment:
Won't this always enable envariant mode on browser?

Reply:
heh, oops. was coding via github UI